image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
@stevvooe that's what I was proposing here https://github.com/docker/docker/pull/26369#discussion_r78636518